crossplane/function/logging.py

```diff
@@ -75,7 +75,7 @@ def dropper(logger, method_name, event_dict):  # noqa: ARG001  # We need this si
             structlog.processors.EventRenamer(to="msg"),
             structlog.processors.JSONRenderer(),
         ],
-        wrapper_class=structlog.make_filtering_bound_logger(logging.INFO),
+        wrapper_class=structlog.make_filtering_bound_logger(logging.DEBUG if level == Level.DEBUG else logging.INFO),
```

Comment:
It should never reach here if `level == Level.DEBUG`, right? See lines 59-67 above.